### PR TITLE
ENG-1562 Bug grafana triggers keyboard short

### PR DESCRIPTION
**Pull request by MikeFuster**

### CODE REVIEW

File: public/app/core/services/keybindingSrv.ts [hunk 1]
```diff
@@ -166,156 +170,158 @@ export class KeybindingSrv {
   }
 
   setupDashboardBindings(dashboard: DashboardModel) {
-    this.bind('mod+o', () => {
```
Comment (bennbollay) [inline public/app/core/services/keybindingSrv.ts]: Consider making the isInAnIframe change in this.bind() instead of everywhere this.bind() is used?

**Review by bennbollay — COMMENTED** [review]
Also make a change to the version file so it pushes a new version.

File: public/app/core/services/keybindingSrv.ts (hunk 1 above)
Comment (MikeFuster) [reply public/app/core/services/keybindingSrv.ts]: Check moved to the bind function

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "license": "AGPL-3.0-only",
   "private": true,
   "name": "grafana",
-  "version": "8.4.0-pre",
+  "version": "8.4.1-pre",
```
Comment (bennbollay) [inline package.json]: Not this version file, the `.fusebit.version.json` file.

File: public/app/core/services/keybindingSrv.ts [hunk 3]
```diff
@@ -132,16 +132,19 @@ export class KeybindingSrv {
...
-      (evt: any) => {
-        evt.preventDefault();
-        evt.stopPropagation();
-        evt.returnValue = false;
-        fn.call(this);
-      },
-      'keydown'
-    );
+    const isInAnIframe = window.location !== window.parent.location;
+    if (!isInAnIframe) {
```
Comment (bennbollay) [inline public/app/core/services/keybindingSrv.ts]: Pro-tip: always lean "exits" towards the smaller branch.

For example:
```
  if (isInAnIframe) {
    return;
  }
```

is a lot more readible than the inversion:

```
  if (!isInAnIframe) {
    Mousetrap.bind(
      keyArg,
      (evt: any) => {
        evt.preventDefault();
        evt.stopPropagation();
        evt.returnValue = false;
        fn.call(this);
      },
      'keydown'
    );
  }
```

Fewer lines indented means that the developer has to hold less state in their head when reading the code, as well.

File: package.json (hunk 2 above)
Comment (MikeFuster) [reply package.json]: Done

File: public/app/core/services/keybindingSrv.ts (hunk 3 above)
Comment (MikeFuster) [reply public/app/core/services/keybindingSrv.ts]: Thanks for the tip, applied!

**Review by fpmanuel — APPROVED** [review]

**Review by bennbollay — APPROVED** [review]

**Pull request merged**